### DISCUSSION TITLE
My college does not provide me an email address and I am not being able to apply for student developer pack

### DISCUSSION THREAD

**Discussion by davidjennicson** [community/community#61773] [GitHub Education]
### Hi everyone,

I am David Jennicson , a final year engineering student . I got a mail today to verify my educational details once again as i availed it for my previous year as well. The email which it suggests is an email id provided only for the faculty members and not the students so what should I do in this case as I will be not eligible for the student developer pack

**Comment by nonvenomous** [comment]
Hey there @DavidJennicson, we and many others are in the same boat.
Unfortunately it seems like we are getting ignored.
Here a link to bigger thread which you should probably upvote and comment on to be notified for any developments, https://github.com/orgs/community/discussions/59360
